test/box/fix-memory-corruption-server.lua

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tarantool
+
+require('console').listen(os.getenv('ADMIN'))
+
+box.cfg({
+    listen = 3301,
+    memtx_memory = 2 * 1024 * 1024 * 1024,
```

Comment:
Why do you need non-default memtx memory?

Reply:
fixed, not need